src/components/document/document-workspace.tsx

```diff
@@ -11,16 +11,21 @@ import {
 } from "../../models/document/document-types";
 import { ImageDragDrop } from "../utilities/image-drag-drop";
 import { NavTabPanel } from "../navigation/nav-tab-panel";
-import { NavTabButtons } from "../navigation/nav-tab-buttons";
+import { ResizePanelDivider } from "./resize-panel-divider";
+import { CollapsedWorkspaceTab } from "./collapsed-workspace-tab";
+import { CollapsedResourcesTab } from "../navigation/collapsed-resources-tab";
 
 import "./document-workspace.sass";
 
 interface IProps extends IBaseProps {
 }
 
+interface IState {
+}
```

Comment:
Is this needed?

Reply:
Strictly not, but I don't mind it as an explicit statement that this component has no state. The behavior is the same without it but the fact that it has no state is implicit rather than explicit.